src/pivot-table/hooks/use-column-width.ts

```diff
@@ -59,13 +71,13 @@ export default function useColumnWidth(
     ...styleService.header,
     bold: true,
   });
+  const { measureText: measureTextForContent, estimateWidth: estimateWidthForContent } = useMeasureText(
+    styleService.content,
+  );
   const { estimateWidth: estimateWidthForRowContent } = useMeasureText(styleService.rowContent);
-  const { measureText: measureTextForColumnContent, estimateWidth: estimateWidthForColumnContent } = useMeasureText(
+  const { estimateWidth: estimateWidthForColumnContent, measureText: measureTextForColumnContent } = useMeasureText(
```

Comment:
in this case I'd say readability is more important than history, 
if you are in middle of some thought process and want to know which variable being renamed to what -> it's so convenient this way rather than trying to chase variables, specially in case like this where names are so similar to each other

Reply:
ah ok, I just thought it was something not reverted from the prev implementation. Let's keep it like this